scripts/openzfs-exceptions.txt

```diff
@@ -86,10 +88,13 @@
 6418 131cc95
 6421 ca0bf58
 6434 472e7c6
+6459 - 			changes don't work properly with ZoL codebase
```

Comment:
I'll make a PR to assign it as WIP and push separate commit with style changes. Thanks for clarification!